### ISSUE TITLE
[Higher Level Review & Supplemental Claims]: Form reloads after radio button choice

### ISSUE THREAD

**Issue by davidakennedy** [department-of-veterans-affairs/va.gov-team#83623]
## What happened?

**Describe what went wrong**:

On the HLR and SC forms, on the first screen: "Is this the form I need?" - When you select a choice for that question, the whole form reloads, and you get the loading indicator. This isn't how the form works elsewhere, so it seems inconsistent and buggy.

**Add the steps to reproduce the bug**:

1. Log in with a test user account. For example: `vetsgov+test236@id.me`.
2. Go to the HLR form, on the first screen: `https://staging.va.gov/decision-reviews/higher-level-review/request-higher-level-review-form-20-0996/start`
3. Select an option, and note how the screen reloads.

**Add a screen recording or photo**:

[Screen recording of Is this the form I need? screen reloading](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBN0g4QkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--423684ef13844652b3dfdbbbd5cde7105723eb97/Screen%20Recording%202024-05-21%20at%201.19.19%E2%80%AFPM.mov)

## How severe is the problem?

- [ ] **Critical**: User is block or significantly hindered from submitting the form, data is lost, or claim and evidence test coverage is missing. Fix ASAP.
- [ ] **Moderate**: User is not blocked, but the happy path is interrupted. Fix after critical bugs.
- [x] **Minor**: Cosmetic bugs and minor language inconsistencies. Defer to a later sprint.

## What is the desired behavior?

1. _When a user selects an option on the screen, the **visual** result is that the screen doesn't load._

## Definition of Done

- [ ] Acceptance criteria reviewed and approved by product or design
- [ ] Unit test (or other test coverage) added if possible
- [ ] Logging and monitoring in place

**Comment by Mottie** [comment]
Work completed and ready for validation in staging.

**Comment by davidakennedy** [comment]
@Mottie Tested on both forms, and everything works as expected!
